test/versioned/kafkajs/kafka.tap.js

```diff
@@ -50,69 +50,78 @@ tap.afterEach(async (t) => {
   await t.context.producer.disconnect()
 })
 
-tap.test('send records correctly', (t) => {
-  t.plan(7)
+try {
```

Comment:
not sure i understand this try/catch. was this meant to be removed after you found your issue?

Reply:
Yep. Debug leftover.